fs/fs.go

```diff
@@ -190,22 +190,30 @@ func getDockerImagePaths(context Context) []string {
 	return dockerImagePaths
 }
 
-// This method compares the mountpoint with possible docker image mount points. If a match is found,
+// This method compares the mountpoints with possible docker image mount points. If a match is found,
 // docker images label is added to the partition.
-func (self *RealFsInfo) updateDockerImagesPath(source string, mountpoint string, dockerImagePaths []string) {
-	for _, v := range dockerImagePaths {
-		if v == mountpoint {
-			if i, ok := self.labels[LabelDockerImages]; ok {
-				// pick the innermost mountpoint.
-				mnt := self.partitions[i].mountpoint
-				if len(mnt) < len(mountpoint) {
-					self.labels[LabelDockerImages] = source
-				}
-			} else {
-				self.labels[LabelDockerImages] = source
+func (self *RealFsInfo) updateDockerImagesPath(mounts []*mount.Info, dockerImagePaths []string) {
+	var useMount *mount.Info
+	for _, m := range mounts {
+		for _, p := range dockerImagePaths {
```

Comment:
nit: To avoid this loop, can we make `getDockerImagePaths()` return a `map[string]struct{}` ?